schemas/rum/resource-schema.json

```diff
@@ -223,6 +223,30 @@
                 }
               },
               "readOnly": true
+            },
+            "graphql": {
+              "type": "object",
+              "description": "GraphQL requests parameters",
+              "required": ["operationType", "query"],
+              "properties": {
+                "operationType": {
+                  "type": "string",
+                  "description": "Type of the GraphQL operation",
+                  "enum": ["query", "mutation", "subscription"],
+                  "readOnly": true
+                },
+                "query": {
```

Comment:
I agree on the `query` naming, I will look for the exact term in the GraphQL docs.

Reply:
I also tend to think it should not be mandatory so that we can at least have the opportunity to drop entirely the content.
I will start an internal conversation to decide on whether to report the payload or not.

Reply:
The Flutter GraphQL client has an issue where it's difficult to get the full payload (because it's already parsed by the time it hits the gql_link). I would appreciate it if the payload was optional.